Xamarin.Forms.Platform.MacOS/Controls/FormsNSSlider.cs

```diff
@@ -0,0 +1,20 @@
+﻿using AppKit;
+using CoreGraphics;
+
+namespace Xamarin.Forms.Platform.macOS.Controls
+{
+	internal class FormsNSSlider : NSSlider
+	{
+		readonly CGSize _fitSize;
+
+		internal FormsNSSlider() : base(CGRect.Empty)
+		{
+			Continuous = true;
+			SizeToFit();
+			var size = Bounds.Size;
+			_fitSize = new CGSize(size.Width > 0 ? size.Width : 100, size.Height > 0 ? size.Height : 20);
```

Comment:
Where did we got that 100x20 ? Is that on Apple documentation ? we should add some comments here about it.

Reply:
@rmarinho frankly speaking, i pull this data from my brain 

```
var size =Bounds.Size; 
// this line returns (0, 19), so, i presume, that 19 is default height
```
but this topic says that default size of thumb is 21x21 
https://discussions.apple.com/thread/4004362
Maybe it makes sense to set default height as 21, but i haven't found any info about default width of these control

Reply:
<img width="920" alt="screenshot at feb 12 22-07-46" src="https://user-images.githubusercontent.com/10124814/52661252-aecb8100-2f12-11e9-8f45-c0dab8bc8ffc.png">

Seems, found :)

http://www.youtube.com/watch?v=JdwVTX9aCtU&t=0m20s

in this video (i attached timecode, 20th sec) that guy uses drag and drop for adding NsSlider, where we can see, that default width = 96 and height 21 indeed.

I will update PR